src/overlays/actors/ovl_Demo_Kankyo/z_demo_kankyo.h

```diff
@@ -6,9 +6,47 @@
 
 struct DemoKankyo;
 
+typedef void (*DemoKankyoActionFunc)(struct DemoKankyo*, GlobalContext*);
+
+typedef enum {
+    /* 0x00 */ DEMOKANKYO_BLUE_RAIN,
+    /* 0x01 */ DEMOKANKYO_BLUE_RAIN_2,
+    /* 0x02 */ DEMOKANKYO_ROCK_1,
+    /* 0x03 */ DEMOKANKYO_ROCK_2,
+    /* 0x04 */ DEMOKANKYO_ROCK_3,
+    /* 0x05 */ DEMOKANKYO_ROCK_4,
+    /* 0x06 */ DEMOKANKYO_ROCK_5,
+    /* 0x07 */ DEMOKANKYO_CLOUDS,
+    /* 0x08 */ DEMOKANKYO_8,        // These unnamed types are removed types that would have used object_gi_melody
+    /* 0x09 */ DEMOKANKYO_9,
+    /* 0x0A */ DEMOKANKYO_A,
+    /* 0x0B */ DEMOKANKYO_B,
+    /* 0x0C */ DEMOKANKYO_C,
+    /* 0x0D */ DEMOKANKYO_DOOR_OF_TIME,
+    /* 0x0E */ DEMOKANKYO_LIGHT_PLANE,
+    /* 0x0F */ DEMOKANKYO_WARP_OUT,
+    /* 0x10 */ DEMOKANKYO_WARP_IN,
+    /* 0x11 */ DEMOKANKYO_SPARKLES
+} DemoKankyo_Type;
+
+typedef struct {
+    Vec3f unk_0;
+    Vec3f unk_C;
+    f32 unk_18;     // For Door of Time, this is the amount to translate it by used for when it's opening
+    f32 unk_1C;
+    s16 unk_20;
+    u8 unk_22;      // mode ?
+    u8 unk_23;
+    s16 unk_24;
```

Comment:
It's not very important but it would be nice to get comments with offsets in this struct